### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by babyhuehnchen**
removed a few duplicates, obsolete/low quality/non-englisch entries and a few entries containing zalgo

### CODE REVIEW

File: web/atlas.json [hunk 1]
```diff
@@ -1644,7 +1636,6 @@
...
-{"id": "twu16a", "submitted_by": "Prudent-Ad5308", "name": "Daft Punk x PNL", "description": "This art is a mix between PNL's album 'Deux frères' and Daft Punk's masks. These two artists are the biggest singer duo in the France music history.\n\nThis artwork was created by Theorus.", "website": "https://twitter.com/Theorus_/status/1258019551373078530?s=20&t=pnJVLlbcZYczhxrB9DaFlQ", "subreddit": "/r/placefrance", "center": [207.5, 1606.5], "path": [[170.5, 1639.5], [244.5, 1640.5], [245.5, 1620.5], [242.5, 1616.5], [244.5, 1609.5], [242.5, 1593.5], [244.5, 1587.5], [241.5, 1581.5], [236.5, 1580.5], [228.5, 1571.5], [216.5, 1566.5], [197.5, 1566.5], [189.5, 1571.5], [182.5, 1573.5], [179.5, 1575.5], [176.5, 1575.5], [173.5, 1580.5], [174.5, 1582.5], [173.5, 1587.5], [171.5, 1592.5], [170.5, 1595.5], [170.5, 1611.5], [173.5, 1618.5], [170.5, 1622.5], [170.5, 1623.5], [170.5, 1637.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:1647]: This entry has no duplicate and is valid for a previous version of the canvas.

File: web/atlas.json [hunk 2]
```diff
@@ -3353,7 +3341,6 @@
...
 {"id": "tx2yi8", "submitted_by": "A_Portuguese_Man", "name": "Azil and Biscoito", "description": "A rendition of Azil's logo and his Hedghog 'Biscoito'.nn", "website": "", "subreddit": "", "center": [999.5, 385.5], "path": [[988.5, 381.5], [1009.5, 381.5], [1009.5, 388.5], [988.5, 388.5], [988.5, 381.5], [988.5, 381.5]]},
-{"id": "tx2yh3", "submitted_by": "TropicalTwister1", "name": "KEL", "description": "KEL is one of the deuteragonists in OMORI. He was relocated multiple times, and his existence can be debated as he was not fully finished right before the WHITE VOID ending of r/place.", "website": "https://omori.fandom.com/wiki/KEL", "subreddit": "/r/omori", "center": [1418.5, 1150.5], "path": [[1415.5, 1136.5], [1415.5, 1136.5], [1422.5, 1136.5], [1426.5, 1141.5], [1427.5, 1146.5], [1429.5, 1148.5], [1428.5, 1149.5], [1423.5, 1155.5], [1426.5, 1158.5], [1424.5, 1162.5], [1423.5, 1162.5], [1422.5, 1167.5], [1420.5, 1168.5], [1419.5, 1164.5], [1416.5, 1164.5], [1416.5, 1168.5], [1413.5, 1167.5], [1412.5, 1161.5], [1410.5, 1161.5], [1410.5, 1159.5], [1409.5, 1158.5], [1413.5, 1155.5], [1408.5, 1150.5], [1407.5, 1147.5], [1408.5, 1142.5], [1413.5, 1137.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:3356]: This entry is valid for a previous version of the canvas.

File: web/atlas.json [hunk 3]
```diff
@@ -3394,7 +3381,6 @@
...
 {"id": "tx2sax", "submitted_by": "Loyso_", "name": "Small bear", "description": "A small bear representing Pinde, a french streamer", "website": "https://www.twitch.tv/pinde", "subreddit": "", "center": [1630.5, 37.5], "path": [[1632.5, 35.5], [1632.5, 41.5], [1626.5, 35.5], [1632.5, 35.5]]},
 {"id": "tx2s2m", "submitted_by": "Gyc3", "name": "Egegge", "description": "Small community that praises Eggdog and eggs. Thanks to Green Lattice for protection", "website": "https://discord.com/invite/NVspkR4", "subreddit": "", "center": [1048.5, 446.5], "path": [[1046.5, 439.5], [1049.5, 439.5], [1052.5, 442.5], [1052.5, 452.5], [1044.5, 452.5], [1043.5, 442.5]]},
-{"id": "tx2s1p", "submitted_by": "Alin_Alexandru", "name": "Second Romanian flag", "description": "The second Romanian flag which featured Brâncuși's Endless Columnn, as well as Spider Man and Miles Morales. Was taken over by Polish streamers.", "website": "", "subreddit": "/r/Romania", "center": [1485.5, 568.5], "path": [[1449.5, 537.5], [1546.5, 537.5], [1546.5, 598.5], [1421.5, 599.5], [1425.5, 541.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:3397]: This entry is valid for a previous version of the canvas.

File: web/atlas.json [hunk 4]
```diff
@@ -4146,7 +4131,6 @@
...
-{"id": "tx63kh", "submitted_by": "link_dead", "name": "2B butt censorship", "description": "During an incursion into the French-controlled corner. North American Twitch streamers xQc and Mizkif deployed their communities to construct a gigantic tribute to the android booty of 2B from Nier: Automata. During the construction of this effigy, Reddit admins censored the image using a gigantic black box. Not deterred, the community again built the effigy to once again be censored by Reddit admins. During construction many members of both North American communities received 15-year bans on placing pixels. Up to 20 percent of Mizkif's 80,000 live viewers were banned during this attack.", "website": "https://en.wikipedia.org/wiki/2B_(Nier:_Automata)", "subreddit": "", "center": [130.5, 1766.5], "path": [[185.5, 1928.5], [183.5, 1602.5], [76.5, 1600.5], [75.5, 1930.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:4149]: This is a valid historical event on the canvas.

File: web/atlas.json [hunk 5]
```diff
@@ -7300,7 +7277,6 @@
...
 {"id": "tyk9je", "submitted_by": "Chiweenies2", "name": "Deutscher Fisch", "description": "A German fish from the city of Rostock.", "website": "", "subreddit": "/r/PlaceFishCult", "center": [291.5, 1626.5], "path": [[289.5, 1625.5], [292.5, 1625.5], [292.5, 1627.5], [289.5, 1627.5]]},
 {"id": "tyk740", "submitted_by": "Chiweenies2", "name": "Chip", "description": "A teal gradient fish named Chip.", "website": "", "subreddit": "/r/PlaceFishCult", "center": [285.5, 1627.5], "path": [[283.5, 1628.5], [283.5, 1626.5], [286.5, 1626.5], [286.5, 1628.5]]},
-{"id": "tyk6ov", "submitted_by": "Andyangel98", "name": "First Romanian Flag", "description": "First Romanian Flag. It was replaced with Polish Flag in the end. The flag contained the Romanian coat of arms, a Dacia car, and other element specific to Romanian countrie.", "website": "", "subreddit": "", "center": [734.5, 179.5], "path": [[636.5, 138.5], [828.5, 139.5], [826.5, 199.5], [785.5, 197.5], [784.5, 240.5], [713.5, 237.5], [711.5, 217.5], [688.5, 198.5], [636.5, 199.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:7303]: This entry is valid for a previous version of the canvas.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Resolve pending comments.

**Comment by ab-gh** [comment]
This PR has been closed. Please reopen when you make the required changes.

**Pull request closed without merging**